src/propTypes.js

```diff
@@ -62,11 +65,14 @@ export const propTypes = {
 }
 
 export const defaultProps = {
+  accessibilityOptions: {
+    closeButtonAriaLabel: 'Close',
```

Comment:
@elrumordelaluz Need some help with this one.

```
accessibilityOptions: PropTypes.shape({
    ariaLabelledBy: PropTypes.string,
    closeButtonAriaLabel: PropTypes.string,
    showNavigationScreenReaders: PropTypes.bool,
  }),
export const defaultProps = {
  accessibilityOptions: {
    closeButtonAriaLabel: 'Close',
    showNavigationScreenReaders: true,
  },
<Component
    accessibilityOptions={{ ariaLabelledBy: 'some-element-id' }}
```
The code above would blow away the default values. Do you know of a clean way to be able to override individual keys in this object without blowing away other default keys?

Reply:
What do you think of something like [this](https://codesandbox.io/s/condescending-kalam-xzdlx?fontsize=14&hidenavigation=1&theme=dark)

Reply:
That totally works. Don't know why I didn't think of that. Thanks!